setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
Thanks @grlee77. I did take a look at PyWavelets and a couple other projects when I was looking into RTD and Cython this weekend. Do you know if any of the docstrings from the Cython code is added to the documentation? I didn't seem to see any, but I could have easily missed it.